### PR TITLE
fix: progress rates were overcounted by 5x

### PR DESCRIPTION
**Pull request by lemmih**
## Summary of changes

Changes introduced in this pull request:

- Divide by `time_in_seconds` to get the real progress in units per second.

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by hanabi1224 — APPROVED** [review]

**Review by LesnyRumcajs — DISMISSED** [review]

File: src/utils/io/progress_log.rs [hunk 1]
```diff
@@ -233,14 +235,14 @@ mod tests {
         progress.last_logged_items = 1024 * 1024 * 1024 / 3;
         assert_eq!(
             progress.msg(now + Duration::from_secs(125)),
-            "test 512 MiB / 1 GiB, 50%, 170.7 MiB/s, elapsed time: 2m 5s"
+            "test 512 MiB / 1 GiB, 50%, 1.4 MiB/s, elapsed time: 2m 5s"
```
Comment (LesnyRumcajs) [inline src/utils/io/progress_log.rs:238]: Could you please explain why the speed is 1.4 MiB/s? It'd be nice to have a comment with the math regarding the expectations.

Comment (lemmih) [reply src/utils/io/progress_log.rs:238]: That's a good point.

File: src/utils/io/progress_log.rs [hunk 2]
```diff
@@ -221,28 +222,30 @@ mod tests {
...
         assert_eq!(
             progress.msg(now + Duration::from_secs(125)),
-            "test 512 MiB / 1 GiB, 50%, 1.4 MiB/s, elapsed time: 2m 5s"
+            "test 512 MiB / 1 GiB, 50%, 3.1 MiB/s, elapsed time: 2m 5s"
         );
 
         progress.set(1024 * 1024 * 1024 / 10);
-        progress.last_logged_items = 1024 * 1024 * 1024 / 11;
+        progress.last_logged_items = 1024 * 1024;
+        // Going from 1MiB to 102.4MiB in 10s should show (1MiB-102.4MiB)/10s = ~10.1 MiB/s
```
Comment (LesnyRumcajs) [inline src/utils/io/progress_log.rs:245]:
```suggestion
        // Going from 1MiB to 102.4MiB in 10s should show (102.4MiB-1MiB)/10s = ~10.1 MiB/s
```

**Review by LesnyRumcajs — APPROVED** [review]

File: src/utils/io/progress_log.rs (hunk 1 above)
Comment (LesnyRumcajs) [reply src/utils/io/progress_log.rs]: Thanks for adding comments, that helps.

**Review by ruseinov — APPROVED** [review]

**Review by hanabi1224 — APPROVED** [review]

**Pull request merged**